src/avgScrabbleScore.js

```diff
@@ -1,4 +1,5 @@
 let scrabbleScore = require('./scrabbleScore');
+console.log(scrabbleScore)
```

Comment:
Remove random debugging `console.log` statements before submitting for feedback.